content/shmem_pe_fence.tex

```diff
@@ -0,0 +1,37 @@
+\apisummary{
+    Assures ordering of delivery of memory store, blocking \PUT{}, 
+    \ac{AMO}, and \OPR{put-with-signal}, as well as nonblocking 
+    \PUT{}, \OPR{put-with-signal}, and \ac{AMO}
+    routines to symmetric data objects issued by the calling \ac{PE} at the target \ac{PE}.
+}
+
+\begin{apidefinition}
+
+\begin{Csynopsis}
+void @\FuncDecl{shmem\_fence}@(int target_pe);
```

Comment:
Is this supposed to be `shmem_pe_fence` here, or is it really `shmem_fence(target_pe)`?